+nosnoc/+dcs/Objects.m

```diff
@@ -0,0 +1,128 @@
+classdef Objects < nosnoc.dcs.Base
+% A particular class of dynamic complementarity system called a gradient complementarity system.
+% It is defined by the dynamics:
+%
+% .. math::
+%     :nowrap:
+%
+%     \begin{align*}
+%        \dot{x} &= f(x) + \nabla c(x)\lambda \\
+%        0 &\le c(x) \perp \lambda \ge 0
+%     \end{align*}
+    properties
+        lambda % casadi.SX|casadi.MX: Lagrange multipliers for the gradient complementarity system.
+        c_lift % casadi.SX|casadi.MX: Lift variables for the gap functions in :class:`nosnoc.model.Pds`.
+
+        x_dot_lift
+        g_friction
+        
+        f_x % casadi.SX or casadi.MX: Right hand side of the gradient complementarity system
+
+        dims % struct: dimensions struct TODO document members
```

Comment:
has everything as the GCS and more, and objects is not a name of a particular DCS.
please explain what is the motivation for this, and better document the class by explaining briefly what is it doing, and what it the difference to GCS